addons/beehave/nodes/composites/composite.gd

```diff
@@ -1,18 +1,29 @@
 ## A Composite node controls the flow of execution of its children in a specific manner.
-extends BeehaveNode
-
-class_name Composite
+@tool
+class_name Composite extends BeehaveNode
 @icon("../../icons/category_composite.svg")
 
 
 var running_child: BeehaveNode = null
 
 
 func _ready():
+	if Engine.is_editor_hint():
+		return
+	
 	if self.get_child_count() < 1:
 		push_error("BehaviorTree Error: Composite %s should have at least one child (NodePath: %s)" % [self.name, self.get_path()])
 
 
+func _get_configuration_warnings() -> PackedStringArray:
+	var warnings: PackedStringArray = super._get_configuration_warnings()
+	
+	if get_children().filter(func(x): return x is BeehaveNode).size() < 2:
```

Comment:
Does this give a warning when `get_children()` is empty?

Reply:
If there are no children `filter` returns empty array, so the condition is still correct.